### PR TITLE
M4768 fullstory

### PR DESCRIPTION
**Pull request by javitonino**
Closes https://github.com/CartoDB/cartodb-management/issues/4768

### CODE REVIEW

**Comment by xavijam** [comment]
No api key needed?

**Comment by javitonino** [comment]
No, just the org ID, which must be added to app_config. I'll document it before deploying.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -217,6 +217,12 @@ def insert_hubspot_form(form = 'newsletter')
     end
   end
 
+  def insert_fullstory
+    if Cartodb.get_config(:fullstory, 'org').present? && current_user && current_user.account_type.casecmp('FREE') == 0
```
Comment (xavijam) [inline app/helpers/application_helper.rb]: Hound rulez!

File: config/app_config.yml.sample [hunk 2]
```diff
@@ -68,6 +68,8 @@ defaults: &defaults
     app_keys:
       editor: ''
       embeds: ''
+  fullstory:
+    org: ''
```
Comment (gfiorav) [inline config/app_config.yml.sample]: Very late shot, but it serves as an example. I think we should start commenting or documenting options that are "off by default" and stop including them equal to `''`. I've recently modified the `fetch_config` method to only return if present, but it feels a bit weird to have this here. Just for the record, anyway.

Comment (xavijam) [inline config/app_config.yml.sample]: I like your idea gfiorav 👍

**Comment by javitonino** [comment]
Some notes for acceptance:
 - Test that the snippet is included and sessions recorded on fullstory panel
 - The snippet should be included only for FREE users
 - Please check that logging in as different users in the same browser (do not use private browsing) works as expected (each sessions is correctly matched to user)
 - Only builder sessions should be logged (no public pages, dashboard, account settings)

File: config/app_config.yml.sample (hunk 2 above)
Comment (javitonino) [inline config/app_config.yml.sample]: It's an easy change, I'll do it along with the NEWS (which I wrote but forgot to commit).

**Pull request merged**